sorc/link_workflow.sh

```diff
@@ -163,15 +165,6 @@ if [[ -d "${script_dir}/gfs_wafs.fd" ]]; then
 fi
 
 
-#------------------------------
```

Comment:
Same question as above.  What happens when a developer does NOT want to use staged GSI fix files?   It may be too cumbersome to have the checkout and link script cover all possibilities.   Perhaps it is sufficient to update the g-w wiki to explain options.